webapp/graphite/metric_filter/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+class MetricFilter(models.Model):
+  id = models.AutoField(primary_key=True)
+  filter = models.CharField(max_length=256)
+  description = models.CharField(max_length=200)
+  type_filter = models.IntegerField(choices=[(0, 'include'), (1, 'exclude')])
+  __str__ = lambda self: "[%s]: %s " % (getTypeFilter(self.type_filter) ,self.filter)
```

Comment:
The best practice in Django land in Python 2 is to define a `__unicode__` method (not `__str__`) and make it return a unicode string. Also it's fine to use `def`.